libs/sdk-flutter/lib/bridge_generated.dart

```diff
@@ -2661,6 +2666,22 @@ class BreezSdkCoreImpl implements BreezSdkCore {
         argNames: ["req"],
       );
 
+  Future<void> rescanSwaps({dynamic hint}) {
```

Comment:
@roeierez This is not exposed through the Flutter plugin, needs to be added to `BreezSDK`(_sdk-flutter/lib/breez_sdk.dart_).